src/components/ForumCard.js

```diff
@@ -0,0 +1,262 @@
+        if (response.status !== 200) throw Error(body.message);
+        return body;
+    }
+
+    //API - Delete your own comments 
+    const handleApiDeleteComment = (commentID) => {
+        callApiDeleteComment(commentID)
+            .then(res => {
+                console.log("callApiDeleteComment returned: ", res)
+            })
+            setTimeout(() => {
+                window.location.reload();
+            }, 100);
+    }
+
+    const callApiDeleteComment = async (commentID) => {
+        const url = `${REACT_APP_API_ENDPOINT}/deleteForumComment`;
+        console.log(url);
+
+        const response = await fetch(url, {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+                commentID: commentID
+            })
+        });
+        const body = await response.json();
+        if (response.status !== 200) throw Error(body.message);
+        return body;
+    }
+
+    return (
+        <div id="body">
+
+            <NavigationBar></NavigationBar>
+
+            <Box sx={{ position: 'absolute', top: 110, left: '50%', transform: 'translate(-50%, -50%)' }}>
+                <Typography
+                    variant="h4"
+                    gutterBottom
+                    component="div">
+                    Forums
+                </Typography>
+            </Box>
+
+            <Box sx={{ position: 'absolute', top: 150, left: '50%', transform: 'translateX(-50%)' }}>
+                {forums.map((event) => (
+                    <Card style={{ width: '1000px', marginBottom: '20px' }} key={event.id}>
+                        <CardContent>
+                            <Link to={`/forum/${event.id}`} target="_blank">
+                                <Typography variant="h5" component="div">
+                                    {event.forumTitle}<br />
+                                </Typography>
+                            </Link>
+                            <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                                Posted on {new Date(event.dateTime).toLocaleDateString()} by {event.creatorName}<br />
+                            </Typography>
+                            <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                                <br />{event.description}<br />
+                            </Typography>
+                            <form onSubmit={handleApiAddComment}>
+                                <TextField
+                                    style={{ mb: 1.5, width: '350px' }}
+                                    label="Add a comment"
+                                    value={newComment}
+                                    onChange={handleNewComment}
+                                />
+                                <Button type="submit" style={{ color: "white", backgroundColor: "seagreen", mb: 1.5 }}>Submit</Button>
```

Comment:
creating a comment does not work, seems to be a front-end issue as the API is never called in my testing. The page refreshes after pressing the button and nothing happens.

Reply:
Not really sure... Would it be possible for you to document the steps you used to test it? So we can recreate the issue, thank you!